### PR TITLE
add compressible and non-compressible se_atten examples

### PR DESCRIPTION
**Pull request by nahso**

### CODE REVIEW

File: examples/water/se_atten/input.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     ],
     "descriptor": {
       "type": "se_atten",
-      "stripped_type_embedding": true,
+      "stripped_type_embedding": false,
```
Comment (wanghan-iapcm) [inline examples/water/se_atten/input.json:10]: new behavior `true` is preferred. @njzjz plz also check

**Comment by njzjz** [comment]
> new behavior `true` is preferred. @njzjz plz also check 

Agree.

File: examples/water/se_atten/input.json (hunk 1 above)
Comment (njzjz) [reply examples/water/se_atten/input.json:10]: > new behavior `true` is preferred. @njzjz plz also check

Agree.

File: examples/water/se_atten_compressible/input.json
Comment (njzjz) [inline examples/water/se_atten_compressible/input.json:1]: This new file should be added to source/tests/test_examples.py to ensure the parameters are always correct.

**Review by njzjz — CHANGES_REQUESTED** [review]

File: examples/water/se_atten_non_compressible/input.json [hunk 2]
```diff
@@ -0,0 +1,87 @@
+{
+  "_comment": " model parameters",
+  "model": {
+    "type_map": [
+      "O",
+      "H"
+    ],
+    "descriptor": {
+      "type": "se_atten",
+      "stripped_type_embedding": false,
```
Comment (njzjz) [inline examples/water/se_atten_non_compressible/input.json:10]: I think @wanghan-iapcm means setting this option to true with 2 attention layers.

**Review by njzjz — APPROVED** [review]

**Comment by wanghan-iapcm** [comment]
close due to #2633

**Pull request closed without merging**